### DISCUSSION TITLE
Welcome to sdk-hsm-sensortag Discussions!

### DISCUSSION THREAD

**Discussion by wassfila** [HomeSmartMesh/sdk-hsm-sensortag#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ksathara** [comment]
Hi guys,

I am new to sensortag developments, so I do apologize if I ask a questions for some of you may sound trivial. 

I have setup Nordic SDK 2.0.0 and I can build sample programs give in the SDK. I want to build the sensortag program. 

I have cloned https://github.com/HomeSmartMesh/sdk-hsm-sensortag.git project.
I tried opening a powershell in the root folder sdk-hsm-sensortag and executing 'west init -m https://github.com/HomeSmartMesh/sdk-hsm-sensortag -mr main' but it failed.
`Traceback (most recent call last):
  File "C:\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\ncs\toolchains\v2.0.0\opt\bin\Scripts\west.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'west'`

What is missing here?

Thanks heaps,
Kay

**Reply by wassfila** [reply]
Hi @ksathara ,
I answered here https://github.com/HomeSmartMesh/sdk-hsm-sensortag/issues/8

You can also open a new discussion or issue as you wish.
